postcss.config.js

```diff
@@ -1,6 +1,10 @@
+const { resolve } = require('path')
+
 module.exports = {
-  plugins: [
-    require('tailwindcss'),
-    require('autoprefixer')
-  ]
+  plugins: {
+    tailwindcss: {
+      config: resolve(__dirname, './tailwind.config.js')
```

Comment:
This is also because when we run the cli, vite will use this file and we want it to use _our_ tailwind config, not the tailwind config of the project that's using this package.